habitat-baselines/habitat_baselines/rl/hrl/hl/neural_policy.py

```diff
@@ -102,28 +101,31 @@ def __init__(
 
         self.aux_modules = get_aux_modules(aux_loss_config, action_space, self)
 
+    def get_termination(
+        self,
+        observations,
+        rnn_hidden_states,
+        prev_actions,
+        masks,
+        cur_skills,
+        log_info,
+    ):
+        if self._config.replan_dist > 0.0:
+            other_agent_dist = observations["other_agent_gps"].norm(dim=-1)
+            return (other_agent_dist < other_agent_dist).cpu()
```

Comment:
did you mean other_agent_dist < self._config.replan_dist?

The current condition will always be false since other_agent_dist == other_agent_dist?

Reply:
Oops, good catch, thanks.